packages/tabs/README.md

```diff
@@ -2,35 +2,74 @@
 
 The `<sp-tabs>` displays a list of `<sp-tab>` element children as `role="tablist"`. An `<sp-tab>` element is associated with a sibling `<sp-tab-panel>` element via their `value` attribute. When an `<sp-tab>` element is `selected`, the associated `<sp-tab-panel>` will also be selected, showing that panel and hiding the others.
 
+[View the design documentation for this component.](https://spectrum.adobe.com/page/tabs/)
+
 ### Usage
 
 [![See it on NPM!](https://img.shields.io/npm/v/@spectrum-web-components/tabs?style=for-the-badge)](https://www.npmjs.com/package/@spectrum-web-components/tabs)
 [![How big is this package in your project?](https://img.shields.io/bundlephobia/minzip/@spectrum-web-components/tabs?style=for-the-badge)](https://bundlephobia.com/result?p=@spectrum-web-components/tabs)
 [![Try it on Stackblitz](https://img.shields.io/badge/Try%20it%20on-Stackblitz-blue?style=for-the-badge)](https://stackblitz.com/edit/vitejs-vite-3smxuwr2)
 
-```
+```zsh
 yarn add @spectrum-web-components/tabs
 ```
 
 Import the side effectful registration of `<sp-tabs>`, `<sp-tab>` or `<sp-tab-panel>` via:
 
-```
+```ts
 import '@spectrum-web-components/tabs/sp-tabs.js';
 import '@spectrum-web-components/tabs/sp-tab.js';
 import '@spectrum-web-components/tabs/sp-tab-panel.js';
 ```
 
 When looking to leverage the `Tabs`, `Tab`, or `TabPanel` base class as a type and/or for extension purposes, do so via:
 
-```
-import {
-    Tabs,
-    Tab,
-    TabPanel
-} from '@spectrum-web-components/tabs';
+```ts
+import { Tabs, Tab, TabPanel } from '@spectrum-web-components/tabs';
 ```
 
-## Sizes
+### Anatomy
+
+- **Tabs:** The container component (`<sp-tabs>`) that manages the tab list and handles selection logic.
+- **Tab item:** An individual tab (`<sp-tab>`) that users can select to view different content panels.
+- **Tab view:** The content panel (`<sp-tab-panel>`) associated with a tab, shown when its corresponding tab is selected.
+- **Divider:** A visual separator between tab items, used in some variants for clarity.
+- **Selection indicator:** A visual highlight (such as an underline or bar) that shows which tab is currently selected.
```

Comment:
````suggestion
- **Selection indicator:** A visual highlight (such as an underline or bar) that shows which tab is currently selected.

```html
<sp-tabs selected="1" size="m">
    <sp-tab label="Tab 1" value="1"></sp-tab>
    <sp-tab label="Tab 2" value="2"></sp-tab>
    <sp-tab label="Tab 3" value="3"></sp-tab>
    <sp-tab label="Tab 4" value="4"></sp-tab>
    <sp-tab-panel value="1">Content for Tab 1</sp-tab-panel>
    <sp-tab-panel value="2">Content for Tab 2</sp-tab-panel>
    <sp-tab-panel value="3">Content for Tab 3</sp-tab-panel>
    <sp-tab-panel value="4">Content for Tab 4</sp-tab-panel>
</sp-tabs>
```
````